uniffi_bindgen/src/bindings/kotlin/templates/EnumTemplate.kt

```diff
@@ -85,14 +85,14 @@ sealed class {{ e.name()|class_name_kt }}{% if e.contains_object_references(ci)
         }.let { /* this makes the `when` an expression, which ensures it is exhaustive */ }
     }
 
-    {% if e.contains_object_references(ci) %}
+    {% if ci.contains_object_references(e) %}
```

Comment:
I'm interested in feedback on this change from a consumer perspective. Rather than asking the enum whether it contains object references and giving it the `ci` for context, we know ask the `ci` whether the given item contains object references. I think it makes things a bit more consistent, especially when you look at the thing that's handling fields below, which was already doing the "ask the `ci`" variant.